### ISSUE TITLE
AoT for datepicker

### ISSUE THREAD

**Issue by lifaon74** [valor-software/ngx-bootstrap#1080]
Trying to build with ngc (last version installed) : 

```
import { DatepickerModule } from 'ng2-bootstrap';
@NgModule({
  imports: [DatepickerModule],
})
export class MyModule {}
```

Results in : 

```
dist\tmp\app\node_modules\ng2-bootstrap\components\datepicker\datepicker.component.ngfactory.ts(24,27): error TS2307: Cannot find module './datepicker-inner.component.ngfactory'.
dist\tmp\app\node_modules\ng2-bootstrap\components\datepicker\datepicker.component.ngfactory.ts(25,27): error TS2307: Cannot find module './daypicker.component.ngfactory'.
dist\tmp\app\node_modules\ng2-bootstrap\components\datepicker\datepicker.component.ngfactory.ts(26,27): error TS2307: Cannot find module './monthpicker.component.ngfactory'.
dist\tmp\app\node_modules\ng2-bootstrap\components\datepicker\datepicker.component.ngfactory.ts(27,27): error TS2307: Cannot find module './yearpicker.component.ngfactory'.
dist\tmp\app\node_modules\ng2-bootstrap\components\datepicker\datepicker.component.ngfactory.ts(178,18): error TS2341: Property 'datePicker' is private and only accessible within class 'DatePickerComponent'.
dist\tmp\app\node_modules\ng2-bootstrap\components\timepicker\timepicker.component.ngfactory.ts(755,42): error TS2445: Property 'updateHours' is protected and only accessible within class 'TimepickerComponent' and its subclasses.
```

Seems Datepicker is not AoT ready ;). Hope it will be soon.

**Comment by Martin-Luft** [comment]
@valorkin datePicker is already public but updateHours is still protected. Same fix as in https://github.com/valor-software/ng2-bootstrap/issues/1079?

**Comment by valorkin** [comment]
Yep

**Comment by Martin-Luft** [comment]
Would be fixed by #1086
